src/Config/spotify.example.js

```diff
@@ -6,7 +6,7 @@
 const spotifyConfig = {
   clientId: "???",
   authEndpoint: 'https://accounts.spotify.com/authorize',
-  redirectUri: "http://localhost:3000/auth",
+  redirectUri: "http://localhost:3000/spotify-callback",
```

Comment:
I will change to the spotify App the allowed redirection path so this is a valid redirect.

Reply:
👍